src/sectionproperties/pre/library/timber_sections.py

```diff
@@ -0,0 +1,66 @@
+"""Timber sections library."""
+
+from __future__ import annotations
+
+import sectionproperties.pre.geometry as geometry
+import sectionproperties.pre.library.primitive_sections as primitive_sections
+import sectionproperties.pre.pre as pre
+
+
+def clt_rectangular_section(
+    d: list[float],
+    lay_orient: list[int],
+    b: float,
+    timb_mat0: pre.Material = pre.DEFAULT_MATERIAL,
+    timb_mat90: pre.Material = pre.DEFAULT_MATERIAL,
+) -> geometry.CompoundGeometry:
+    """Constructs a timber rectangular section.
+
+    Constructs a timber rectangular section of depth ``d`` and width ``b``.
+
+    .. note::
+
+    Args:
+        d: Timber layer section thickness
+        lay_orient: List of layer orientation
+        b: Timber section width
+        timb_mat0: Material object to assign to the timber area
+                   parallel-to-grain
+        timb_mat90: Material object to assign to the timber area,
+                    perpendicular-to-grain
+
+    Raises:
+        ValueError: Geometry generation failed
+
+    Returns:
+        Timber rectangular section geometry
+
+    Example:
+        The following example creates a 120mm CLT cross-section.
+    """
+
+    layer_geom = list()
+    for idx in range(len(d)):
+        di = float(d[idx])
+        layer = lay_orient[idx]
+
+        if layer is int(0):
```

Comment:
This seems redundant and misses the mark slightly. Could it not just be `if layer == 0`?